regexes/device/mobiles.yml

```diff
@@ -2930,6 +2930,14 @@ Asus:
       device: 'desktop'
       model: 'Chromebook Flip $1'
 
+# Auchan (https://www.auchan.lu/)
+Auchan:
+  regex: 'S4T7IN(?:[);/ ]|$)'
+  device: 'tablet'
+  models:
+    - regex: 'S4T7IN(?:[);/ ]|$)'
+      model: 'Selecline S4T7IN'
```

Comment:
rename brand Auchan to Selecline

`Selecline` is a trademark of the retailer (Auchan).
`Qilive` is a trademark of the retailer (Auchan).